pipelines/LsstCamImSim/ProcessCcdCalibrateImage.yaml

```diff
@@ -0,0 +1,27 @@
+description: Raw image pipeline specialized for ImSim.
+instrument: lsst.obs.lsst.LsstCamImSim
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcdCalibrateImage.yaml
+
+tasks:
+  isr:
+    class: lsst.ip.isr.IsrTask
+    config:
+      connections.newBFKernel: bfk
+      doDefect: False
+      doBrighterFatter: True
+  calibrateImage:
+    class: lsst.pipe.tasks.calibrateImage.CalibrateImageTask
+    config:
+      # HACK: workaround for DM-31047, remove once we're no longer loading
+      # DRP-specific obs configs.
+      file: $AP_PIPE_DIR/config/LSSTCam-imSim/calibrateImage.py
+      # Do not integrate file and pipeline configs until DM-31047 fixed, to
+      # make it easier to check for changes on the obs side.
+      # Config file wipes out all pre-existing configs, so copy base pipeline
+      # config on top.
+      connections.astrometry_ref_cat: 'cal_ref_cat_2_2'
+      connections.photometry_ref_cat: 'cal_ref_cat_2_2'
+      python: >
+        config.astrometry_ref_loader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
+        config.photometry_ref_loader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
```

Comment:
This is a new task, and not susceptible to DM-31047 (especially since `calibrateImage.py` doesn't have the "reset to task default" hack).